File: buzzer.py
import RPi.GPIO as GPIO
from time import sleep

buzzer_pin = 23

GPIO.setwarnings(False)
GPIO.setmode(GPIO.BCM)
GPIO.setup(buzzer_pin, GPIO.OUT, initial=GPIO.LOW)

# ketika output altitude dari bmp 388 bernilai 0
# maka buzzer akan menyala
while (some_altitude_function == 0):
    GPIO.output(buzzer_pin, GPIO.HIGH)
    sleep(5) # buzzes for 5 secs
    GPIO.output(buzzer_pin, GPIO.LOW)
    
GPIO.cleanup()

File: CODE YANG BARU (TEENSY)/buzzer.cpp
int buzzerPin = 13;

void setup() {
  Serial.begin(115200);
  pinMode(buzzerPin, OUTPUT);
}

void loop() {
  if (some_altitude_function() == 0) {
    // buzzes for 5 seconds 
    digitalWrite(buzzerPIN, HIGH);
    delay(5000);
    digitalWrite(buzzerPIN, LOW);
    delay(2000);
  }
}

// placeholder function
int some_altitude_function() {
  return 0;
}
